### DISCUSSION TITLE
Do I need to install Lua 5.1 when Neovim has luaJIT embedded?

### DISCUSSION THREAD

**Discussion by andrewcrook** [lumen-oss/rocks.nvim#309] [Q&A] [answered]
- On macOS Homebrew has installed Lua 5.4 and LuaRocks as a dependency for another formula.
- So I would like to have neovim to use its own dedicated lua 5.1 setup. I only use this for configuration and plan to use fennel.
- Therefore, I would like to install a version of LuaRocks from within neovim.
- Here’s the question do I need to install lua 5.1 when neovim has luajit (based on lua 5.1) embedded?

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

Afaik, yes, you do need to have lua 5.1 installed, because luarocks will fail to install rocks for version 5.1 if it can't find a lua 5.1 installation. And it won't recognize neovim as a lua 5.1 installation. 

I believe @vhyrro has plans to try and find a way to get luarocks to recognise neovim.

**Reply by andrewcrook** [reply]
Thanks for varifying
